source/util/AMSettings.cpp

```diff
@@ -171,9 +170,9 @@ void AMSettings::load() {
 	publicDataFolder_ = settings.value("publicDataFolder", "/home/acquaman/data/").toString();
 	publicDatabaseFilename_ = settings.value("publicDatabaseFilename", "publicdata.db").toString();
 
-	QString defaultBasePath(QDir::homePath() % "/beamline/programming");
-	fileLoaderPluginsFolder_ = settings.value("fileLoaderPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/FileLoaders")).toString();
-	analysisBlockPluginsFolder_ = settings.value("analysisBlockPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/AnalysisBlocks")).toString();
+	QString defaultBasePath(QCoreApplication::applicationDirPath() % "/..");
```

Comment:
Rather than just adding in `/..` we could throw the string from `QCoreApplication::applicationDirPath()` into a QFileInfo, then use `cdUp()` (synonymous with with `cd ..` then check the string is correct.  Thoughts?